python/cudf/cudf/utils/applyutils.py

```diff
@@ -174,7 +177,7 @@ def run(self, df, **launch_params):
             )
             if out_mask is not None:
                 outdf._data[k] = outdf[k]._column.set_mask(
-                    out_mask.data_array_view
+                    out_mask.data_array_view(mode="write")
```

Comment:
So `out_mask` is just created, so this is just an indication that we are taking ownership of this object, I guess.

Reply:
Right